includes/abstracts/abstract-wc-product.php

```diff
@@ -549,6 +550,17 @@ public function get_menu_order( $context = 'view' ) {
 		return $this->get_prop( 'menu_order', $context );
 	}
 
+	/**
+	 * Get post password.
+	 *
+	 * @since  3.6.0
+	 * @param  string $context What the value is for. Valid values are view and edit.
+	 * @return int
+	 */
+	public function get_post_password( $context = 'view' ) {
```

Comment:
Probably not an issue, but a `view` context for post password . . . 🤔

Reply:
Just for consistency. As you know, view allows filtering.